plugins/remark.ts

```diff
@@ -27,6 +27,9 @@ export interface Options {
   /** List of rehype plugins to use */
   rehypePlugins?: unknown[];
 
+  /** Configuration for remark-rehype */
+  remarkRehype?: object;
```

Comment:
I'd change this option to `remarkOptions` object for consistency with `rehypePlugins` and to make clear that it contains the options for `rehype`, not the `remarkRehype` option itself.

Reply:
Ah ok, I wasn't quite clear on whether remark-rehype was still technically a remark plugin or if it was a standalone thing of sorts but yeah, that makes more sense